features/netsocket/cellular/cellular_driver_l3ip.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2013 - 2014, Freescale Semiconductor, Inc.
+ * Copyright (c) 2017 ARM Limited
+ *
+ * All rights reserved.
```

Comment:
This file might be under e.g. some Greentea test like in https://github.com/ARMmbed/mbed-os/tree/master/TESTS/network/emac where there is memory manager and test network stacks.

Reply:
done